CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
+## [0.4.3] - 2024-05-09
```

Comment:
no need for a new release, probably can even add it to 0.4.1 since we haven't even released that one yet.